api_tests/npm_wrapper.sh

```diff
@@ -7,5 +7,5 @@ export TEST_DIR="$1";
 cd "${ROOT}/api_tests";
 yarn install;
 
-npm test "./harness.js" \
+node "./harness.js" \
```

Comment:
:wrench: You should be able to call it directly right?

```suggestion
"./harness.js" \
```

Reply:
err wait didn't you delete this thing? Do we still need this for anything?

Reply:
It's deleted.